src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/OppfolgingstilfelleBitService.kt

```diff
@@ -39,6 +46,19 @@ class OppfolgingstilfelleBitService() {
         }
     }
 
+    private fun handleDuplicate(
+        connection: Connection,
+        existing: OppfolgingstilfelleBit,
+        incoming: OppfolgingstilfelleBit,
+    ) {
+        if (existing.uuid == incoming.uuid && existing.korrigerer == null && incoming.korrigerer != null) {
```

Comment:
Kunne vært kult med infix funksjon på bit her ala `if (incoming korrigerer existing)` 🤓

Reply:
Akkurat i denne konteksten er det jo bare for å rette opp et felt som ikke ble persistert i første omgang, men kanskje en ide for der vi faktisk bruker korreksjonen (det er i List<OppfolgingstilfelleBit>.filtrerOppfolgingstilfelleList())